sdk/eventhub/azure-eventhub/stress/scripts/azure_eventhub_producer_stress.py

```diff
@@ -169,27 +171,39 @@ def create_client(self, client_class, is_async=False):
                 "password": self.args.proxy_password,
             }
 
-        if self.args.conn_str:
-            client = client_class.from_connection_string(
-                self.args.conn_str,
+        if self.args.azure_identity:
+            print("Using Azure Identity")
+            client = client_class(
+                fully_qualified_namespace=self.args.hostname,
                 eventhub_name=self.args.eventhub,
+                credential=DefaultAzureCredential(),
                 auth_timeout=self.args.auth_timeout,
                 http_proxy=http_proxy,
                 transport_type=transport_type,
                 logging_enable=self.args.uamqp_logging_enable,
-                **retry_options
+                **retry_options 
```

Comment:
nit probably doesn't matter:
```suggestion
                **retry_options
```